geoutils/georaster.py

```diff
@@ -69,6 +69,31 @@ def _resampling_from_str(resampling: str) -> Resampling:
     return resampling_method
 
 
+# Function to set the default nodata values for any given dtype
+# Similar to GDAL for int types, but without absurdly long nodata values for floats.
+# For unsigned types, the maximum value is chosen (with a max of 99999).
+# For signed types, the minimum value is chosen (with a min of -99999).
+def default_ndv(dtype: str) -> int:
```

Comment:
Could this be hidden from the public interface, aka: `_default_ndv()`?

Reply:
Or it won't make much of a difference if `Raster` is moved, come to think of it.

Reply:
Hidden in ae955e5 anyway.